ml-libs/CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ if(THEROCK_ENABLE_COMPOSABLE_KERNEL)
     RUNTIME_DEPS
       hip-clr
       rocm-half
-      rocRAND
+      ${optional_rand_deps}
```

Comment:
`optional_rand_deps` is defined further down in the file. Is that okay? Maybe reorder? Move up next to where `optional_profiler_deps` is defined?

Could also prefix local variables with `_`